scripts/analytics/institutions.py

```diff
@@ -0,0 +1,53 @@
+from modularodm import Q
+
+from website.app import init_app
+from website.models import User, Node, Institution
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+
+def get_institutions():
+    institutions = Institution.find(Q('_id', 'ne', None))
+    return institutions
+
+
+def get_count_by_institutions():
+    institutions = get_institutions()
+    counts = []
+
+    for institution in institutions:
+        user_query = Q('_affiliated_institutions', 'eq', institution.node)
+        node_query = (
+            Q('is_deleted', 'ne', True) &
+            Q('is_folder', 'ne', True) &
+            Q('parent_node', 'eq', None)
+        )
+        count = {
+            'institution': institution.name,
+            'users': User.find(user_query).count(),
+            'nodes': Node.find_by_institutions(institution, node_query).count(),
```

Comment:
For these two, we might want to add breakdowns later, so I would suggest an extra layer of nesting:
````